app/views/shared/_title_bar.html.erb

```diff
@@ -1,13 +1,13 @@
         <div class="container">
           <div id="title-bar" class="row">
-            <div id="site-search" class="col-md-6">
+            <div id="site-search" class="col-md-10">
               <%= render 'shared/site_search' if show_site_search? %>
             </div>
-
+	    <!--
```

Comment:
There's no reason to comment this, may as well just remove it. We can always get it back from the git history.